src/components/TodoInput/TodoInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInput: React.FC<Props> = ({ todo }) => (
```

Comment:
it's not input (there could be input during editing todo, but the entire component is not just input). It's one item of the TodoList, so let's reneme it to TodoItem

```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => (
```